teamspeak3.kvp

```diff
@@ -0,0 +1,74 @@
+Meta.DisplayName=TeamSpeak 3
+Meta.Description=TeamSpeak 3 Server
+Meta.OS=Windows, Linux
+Meta.Author=IceOfWraith, YamiKaitou (base)
+Meta.URL=https://teamspeak.com
+Meta.DisplayImageSource=url:https://www.teamspeak.com/user/themes/teamspeak/images/logo_inverse.svg
+Meta.EndpointURIFormat=steam://connect/{0}
+Meta.ConfigManifest=teamspeak3-config.json
```

Comment:
Convention is not to include the dash for config.json and metaconfig.json

Reply:
Convention be damned.